src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Startup.cs

```diff
@@ -49,7 +49,7 @@ public static void Configure(IApplicationBuilder app, IHostingEnvironment env)
             }
             else
             {
-                app.UseExceptionHandler("/Home/Error");
+                app.UseStatusCodePagesWithRedirects("/Error?sc={0}");
```

Comment:
maybe consider renaming _sc_ to _statusCode_?

Reply:
Renamed sc to statusCode